app/assets/javascripts/app/payment.js

```diff
@@ -42,7 +42,13 @@ App.addChild('Payment', _.extend({
   },
 
   selectNationalPayment: function() {
-    this.onTabClick({currentTarget: this.$('#MoIP')});
+    $tab_target = $('#payment-methods').data('preferred-engine');
```

Comment:
You need a `var` here before the variable name, otherwise you'll create a global. Also, use camelCase 😄 

For the argument of `data`, you can also use camelCase and it will work (I mean, `data('preferredEngine')` will get the value for `data-preferred-engine`).